docs/camino-node/gcloud.md

```diff
@@ -0,0 +1,176 @@
+---
+sidebar_position: 10
+title: Run Camino Node with Google Cloud Platform
+description: This guide provides step-by-step instructions on how to set up and run Camino Node on the Google Cloud Platform.
+---
+
+# Setting Up Camino Node on Google Cloud Platform
+
+This tutorial will walk you through the process of configuring a Camino Node on the Google Cloud Platform (GCP).
+Starting with creating a firewall rule to open the necessary port, setting up a virtual machine (VM) instance,
+and configuring Camino Node to operate seamlessly on the Google Cloud Platform. Whether you are a seasoned
+developer or a beginner exploring blockchain technology, this guide is tailored to provide clear instructions
+and ensure a smooth setup process.
+
+Before we dive into the technical details, it's essential to understand the significance of the staking and API ports used
+by Camino Node. We will also touch upon the option of exposing your node as an API node for further utility. By the end of
+this guide, you'll have a fully operational Camino Node, ready to contribute to the decentralized network and participate
+in the Camino ecosystem.
+
+Let's get started with the journey of setting up your Camino Node on Google Cloud Platform!
+
+## Create Firewall Rule to Open Port 9650
+
+1. Access the Google Cloud Platform (GCP) Console and navigate to "VPC Network" > "Firewall."
+
+<figure>
+<img class="zoom" src="/img/gcloud/gcloud_10.png#center"/>
+<figcaption align = "center"><b>Fig.1:</b> Click "VPC Network" and select "Firewall"</figcaption>
+</figure>
+
+2. Click on "Create Firewall Rule" and set it to allow TCP port 9650 for all inbound traffic. Set the target tag to be "camino-network-p2p."
+
+<figure>
+<img class="zoom" src="/img/gcloud/gcloud_11.png#center"/>
+<figcaption align = "center"><b>Fig.2:</b> Click "Create Firewall Rule"</figcaption>
+</figure>
+
+3. Configure the firewall rule as shown in the screenshots below:
+
+<figure>
+<img class="zoom" src="/img/gcloud/gcloud_12.png#center"/>
+<figcaption align = "center"><b>Fig.3:</b> Configure Firewall rule</figcaption>
+</figure>
+
+4. Enter "9650" for TCP ports under "Specified ports and protocols." Set the other fields as shown in the screenshot below and click "Create".
+
+:::tip UNDERSTANDING PORTS: STAKING & API PORTS
+
+Camino Node uses two ports for different features. Port 9650 is the staking port, used to communicate with
+the validator nodes on the Camino Network. There is another port, 9651, used for exposing the API. If you
+want to use your node as an API node, you can also add port 9651 to the allowed ports in this step.
+
+For more information about APIs, please refer to the [Camino Node APIs](/developer/apis/camino-node-apis/) documentation.
+
+:::
+
+<figure>
+<img class="zoom" src="/img/gcloud/gcloud_13.png#center"/>
```

Comment:
I'm sorry the screenshot is wrong, I will send you a new one. it should be 9651 instead of 9650

Reply:
Fixed